### PR TITLE
BlueScreen: add renderToFile

### PR DESCRIPTION
**Pull request by Vrtak-CZ**
We are using Tracy with [Monolog](https://github.com/Seldaek/monolog) and we only need "render" bluescreen to file. So I add renderToFile method to BlueScreen (code moved from Logger).

I'm not sure with method name. You maybe have better idea.

### CODE REVIEW

**Comment by fprochazka** [comment]
Nice :+1:

File: src/Tracy/BlueScreen.php [hunk 1]
```diff
@@ -68,6 +68,24 @@ public function render($exception)
 
 
 	/**
+	 * @param \Exception|\Throwable
+	 * @param string file path
+	 * @return void
+	 */
+	public function renderToFile($exception, $file)
+	{
+		if ($handle = @fopen($file, 'x')) { // @ file may already exist
```
Comment (dg) [inline src/Tracy/BlueScreen.php]: IMHO this conditions should stay in Logger.

Comment (Vrtak-CZ) [inline src/Tracy/BlueScreen.php]: Ok but if I move that to Logger I must pass resource to this method from Logger or we should have "copy&paste" code in Logger and in this method. But file as resource for this method doesn't looks good for me (it's just feeling).

Do you have better idea?

Comment (dg) [inline src/Tracy/BlueScreen.php]: File should be opened twice, via `@fopen($file, 'x')` in logger and `fopen($file, 'w')` in Bluescreen.

**Pull request closed without merging**